### PR TITLE
MPDX-9847,MPDX-9848,MPDX-9849 - NSO MPD Questionnaire mobile view fixes, move phone number fields to Staff Info card, remove sidepanel

### PR DESCRIPTION
**Pull request by zweatshirt**
## Description

- With the current design, labels on smaller screens get truncated and are unreadable on mobile views.
- This PR fixes the issue by setting labels above input fields instead of inside of them.
- This also fixes the Back and Continue button on small screens, as well as the Phone Number input fields.

## Testing

- Go to the NSO MPD Questionnaire
- Determine if the design still looks okay, and test the mobile view.

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels (_Add the label "Preview" to automatically create a preview environment_)
- [x] I have run the Claude Code `/pr-review` command locally and fixed any relevant suggestions
- [ ] I have requested a review from another person on the project
- [x] I have tested my changes in preview or in staging
- [x] I have cleaned up my commit history

### CODE REVIEW

File: src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.tsx [hunk 1]
```diff
@@ -48,99 +49,155 @@ export const MinistryDetails: React.FC = () => {
...
+    helperText: cityHelperText,
+    ...cityFieldProps
+  } = useQuestionnaireAutoSave({
     fieldName: 'geographicLocation',
     schema,
     saveOnChange: true,
   });
 
+  // The ministry select surfaces a load failure in place of any autosave error.
+  const ministryEffectiveError =
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.tsx]: [Medium] **Untested refactored branch.** This PR reworked the ministry load-failure path into `ministryEffectiveError` / `ministryEffectiveHelperText`, but `MinistryDetails.test.tsx` only covers the *loading* state — there is no test for the *unavailable* state (empty OneApp list → `ministriesUnavailable` → 'Failed to load ministries' + error). Since `ministriesUnavailable = !ministriesLoading && ministries.length === 0`, an empty result drives it. Add a test that renders with an empty ministries result and asserts the failure message + the select's error/disabled state, so this branch can't silently regress. _(Testing agent)_

File: src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.tsx [hunk 2]
```diff
@@ -48,99 +49,155 @@ export const MinistryDetails: React.FC = () => {
...
+            select
+            required
+            error={ministryEffectiveError}
+            size="small"
+            slotProps={{
+              select: {
+                displayEmpty: true,
+                labelId: aria['aria-labelledby'],
+              },
+              htmlInput: { 'aria-describedby': aria['aria-describedby'] },
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.tsx]: [Suggestion] **Select helper-text a11y (please verify).** For the two `<TextField select>` fields, `aria-describedby` is placed on `slotProps.htmlInput` — which targets the Select's *hidden* native input, not the focusable `role=combobox` element. So a screen reader may not announce the helper text (e.g. the city field's guidance). The accessible *name* via `slotProps.select.labelId` is correct. Verify with a screen reader; if it doesn't announce, also set `slotProps.select['aria-describedby']`. Same pattern on the city select (~line 177). _(UX + Architecture + Data-Integrity agents)_

File: src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.tsx [hunk 3]
```diff
@@ -48,99 +49,155 @@ export const MinistryDetails: React.FC = () => {
...
+        )}
+      </LabeledField>
 
       {loading ? (
         <CircularProgress />
       ) : (
-        <TextField
-          select
-          required
+        <LabeledField
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.tsx]: [Medium] **Duplicated select body.** The ministry and city selects now share ~18 near-identical lines (`slotProps.select`/`htmlInput`/`input` + the empty placeholder `MenuItem` + the options `.map`). Consider extracting a `SelectQuestion` sibling to `RadioQuestion` / `NumberQuestion` so the two selects can't drift (e.g. an aria fix applied to one and not the other). Net debt is still reduced by this PR — reasonable as a follow-up. _(Architecture agent)_

File: src/components/HrTools/NsoMpdQuestionnaire/FinancialInformation/DebtQuestions.tsx [hunk 4]
```diff
@@ -108,25 +106,31 @@ export const DebtQuestions: React.FC = () => {
...
-          value={hasDebt}
-          onChange={handleHasDebtChange}
-          onBlur={() => setHasDebtTouched(true)}
-        >
-          <FormControlLabel value="Yes" control={<Radio />} label={t('Yes')} />
-          <FormControlLabel value="No" control={<Radio />} label={t('No')} />
-        </RadioGroup>
-        {showHasDebtError && (
-          <FormHelperText>{t('Please select an answer.')}</FormHelperText>
+      <LabeledField
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/FinancialInformation/DebtQuestions.tsx:109]: [Suggestion] **Missing `required` affordance.** The has-debt question is effectively required (it validates and blocks the step) but this `LabeledField` isn't passed `required`, so it shows no asterisk — unlike `RadioQuestion` and the other questions. (Mirrors pre-existing behavior, but the block was rewritten here.) Pass `required` for parity. _(UX + Architecture agents)_

File: src/components/HrTools/NsoMpdQuestionnaire/Shared/LabeledField.test.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+import { render } from '@testing-library/react';
+import { LabeledField } from './LabeledField';
+
+interface RenderOptions {
+  required?: boolean;
+  error?: boolean;
+  helperText?: React.ReactNode;
+}
+
+const renderField = ({ required, error, helperText }: RenderOptions = {}) =>
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/Shared/LabeledField.test.tsx:11]: [Suggestion] **Test convention.** This suite uses a `renderField(...)` helper, while the sibling suites added in this same PR (`DebtQuestions`, `MinistryDetails`) use the `TestComponent` + `render(<TestComponent />)` pattern — the project's preferred convention. Consider a `TestComponent` wrapper for consistency. _(Standards + Testing agents)_

File: src/components/HrTools/NsoMpdQuestionnaire/Shared/LabeledField.test.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    );
+  });
+
+  it('marks the label required', () => {
+    const { getByText } = renderField({ required: true });
+
+    expect(getByText('Question text')).toHaveClass('Mui-required');
+  });
+
+  it('reflects the error state on the label', () => {
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/Shared/LabeledField.test.tsx:62]: [Suggestion] **Strengthen the error test.** This asserts the `Mui-error` class but not that the passed `helperText` ('Bad') actually renders while in error — which is the real-world use (consumers pass the validation message as `helperText` when `error`). Add `expect(getByText('Bad')).toBeInTheDocument()` (and optionally that it's still wired via `aria-describedby`). _(Testing agent)_

File: src/components/HrTools/NsoMpdQuestionnaire/Shared/NsoMpdQuestionnaireLayout.tsx [hunk 7]
```diff
@@ -79,7 +79,7 @@ export const NsoMpdQuestionnaireLayout: React.FC<
           <Stack spacing={4}>
             {mainContent}
             {!isLastStep && (
-              <Stack direction="row" spacing={2} mx={4}>
+              <Stack direction={{ xs: 'column', sm: 'row' }} spacing={2} mx={4}>
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/Shared/NsoMpdQuestionnaireLayout.tsx:68]: [Suggestion] **Mobile button order (design call).** In the new column stack, Back renders above the primary Continue action (same on Summary's Submit row). Some mobile patterns lead with — or bottom-anchor — the primary CTA for thumb reach. Confirm the intended order with design; `flexDirection: { xs: 'column-reverse', sm: 'row' }` would put the primary action first on mobile. Low priority. _(UX agent)_

**Review by zweatshirt — COMMENTED** [review]
## 🤖 Multi-Agent Review — APPROVED WITH SUGGESTIONS ✅

6 specialized agents (Architecture, Data Integrity, Testing, UX, Standards, Financial Reporting) reviewed this PR. **No blockers.** A well-executed presentational/responsive refactor — one tested `LabeledField` abstraction replaces four bespoke label structures (net tech-debt reduction), the MUI v7 `slotProps` migration is correct, data-save wiring is provably intact, and the full questionnaire suite is green (172/172, ESLint clean).

**Risk:** substantively **LOW** — no auth / API / GraphQL / schema / money-logic changes, and the blast radius is contained to the NSO questionnaire feature. (The mechanical file-count risk score is inflated purely by touching 12 component files.)

**Suggestions — all severity < 7, none blocking (inline below):**
- 🟡 Add a test for the `MinistryDetails` ministries-unavailable failure branch — it was reworked in this PR and is currently untested. *The one worth doing before merge.*
- 🟡 Verify the two selects' `aria-describedby` reaches the focusable combobox (helper-text a11y); one-line fix if it regresses.
- 🔵 Optional polish: extract a `SelectQuestion` to de-dupe the two select bodies; add `required` to the has-debt question; align `LabeledField.test` with the `TestComponent` convention; confirm the mobile Back/Continue stack order.

Financial Reporting agent: _“No financial calculation code in this PR.”_ Security agent: skipped (no auth/API/config surface). Layout-only files (dividers, responsive `direction`) can only be verified in a browser/preview — jsdom can't assert breakpoints.

File: src/components/HrTools/NsoMpdQuestionnaire/Shared/QuestionnaireActionButton.tsx [hunk 8]
```diff
@@ -13,6 +13,11 @@ const StyledButton = styled(Button)(({ theme }) => ({
   paddingInline: theme.spacing(4),
   paddingBlock: theme.spacing(1),
   minWidth: theme.spacing(20),
+  // On small screens the buttons stack into a column , so let
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/Shared/QuestionnaireActionButton.tsx:16]: [Suggestion] Nit: stray space before the comma — `stack into a column , so let` → `stack into a column, so let`.

File: src/components/HrTools/NsoMpdQuestionnaire/PersonalInformation/StaffInformation.tsx [hunk 9]
```diff
@@ -96,9 +103,12 @@ export const StaffInformation: React.FC = () => {
               key={field.label}
               label={field.label}
               value={field.value}
-              // readOnly instead of disabled purely for accessibility
-              InputProps={{ readOnly: true }}
+              placeholder={t('Not on record')}
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/PersonalInformation/StaffInformation.tsx:106]: [Suggestion] The "Not on record" empty-state is conveyed only via the `placeholder`, whose screen-reader announcement is inconsistent across AT and whose default contrast is muted. Fine for a read-only card (the `label` always carries the field name), so no change is required — flagging the tradeoff. Also consider re-adding the one-line rationale that `readOnly` (not `disabled`) is intentional so the fields stay focusable/announced — that note was dropped in the `slotProps` migration. _(UX agent)_

File: src/components/HrTools/NsoMpdQuestionnaire/NsoMpdQuestionnaireTestWrapper.tsx [hunk 10]
```diff
@@ -36,13 +36,29 @@ export const newStaffQuestionnaireMock: DeepPartial<
   creditCardDebtMonthlyPayment: null,
 };
 
+type MinistryMock = { name: string; children: { name: string }[] };
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/NsoMpdQuestionnaireTestWrapper.tsx:39]: [Suggestion] The hand-rolled `MinistryMock` type diverges from the sibling `newStaffQuestionnaire` prop (typed via generated types with `DeepPartial`). It won't track drift if the `Ministries` selection set gains fields. Consider `DeepPartial<MinistriesQuery['ministries'][number]>[]` (or `MinistriesQuery['ministries']`) to reuse generated types. Non-blocking. _(Testing agent)_

File: src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.test.tsx [hunk 11]
```diff
@@ -137,4 +137,27 @@ describe('MinistryDetails', () => {
...
+
+  it('renders the assignment-location question as a label above the field', () => {
+    const { getByText } = render(<TestComponent />);
+
+    expect(
+      getByText('What is your expected ministry assignment location?').tagName,
+    ).toBe('SPAN');
+  });
+
+  it('surfaces a load failure when no ministries are available', async () => {
```
Comment (zweatshirt) [inline src/components/HrTools/NsoMpdQuestionnaire/MinistryInformation/MinistryDetails.test.tsx:149]: [Suggestion] This test renders `NsoMpdQuestionnaireTestWrapper` inline (to pass `ministries={[]}`) rather than through the file's `TestComponent` helper, which currently only threads `onCall`. Minor consistency nit — optionally extend `TestComponent` to forward a `ministries` prop so all tests share one render path. _(Testing agent)_

**Review by zweatshirt — COMMENTED** [review]
## 🤖 Multi-Agent Review (focused re-review) — CLEAN ✅

Focused pass on the **newest delta only** — the two review-driven fixes since the prior review (the ministries-unavailable test and the "Not on record" placeholder). The rest of this PR was already covered by two earlier multi-agent reviews. 2 agents (Testing + UX) reviewed the ~40-line delta; the other specializations had nothing in scope.

**Verdict: CLEAN** — no blockers, no important issues. Both prior-review items are now **resolved**:
- ✅ `MinistryDetails` ministries-unavailable branch now has a regression test (empty OneApp list → disabled + "Failed to load ministries").
- ✅ Empty read-only Staff Info fields now show a "Not on record" placeholder (card-wide).

Suite green (15/15 on the affected specs); the `slotProps` migration is correct MUI v7 (ahead of surrounding code that still uses the deprecated API). The inline comments below are all **severity < 5 (informational)** — tradeoff notes / consistency nits, nothing that blocks merge.

One UX tradeoff worth noting (no change required): the "Not on record" state is carried only by the HTML `placeholder`, whose screen-reader announcement varies across AT and whose default contrast is muted. Acceptable for a read-only verification card since each field always carries its name via the `label`; a filled sentinel value would announce more reliably but reads more like real data.
